### PR TITLE
SRE-2627 Add support for docker multi-arch

### PR DESCRIPTION
**Pull request by JesperTerkelsen**
**Motivation** With the [graviton2](https://aws.amazon.com/ec2/graviton/) potential cost savings, and [Apple silicon](https://en.wikipedia.org/wiki/Apple_silicon) being the only option for local development soon, we need to support running parts of our infrastructure on [ARM](https://en.wikipedia.org/wiki/ARM_architecture) based architectures.

See here for how this works: https://docs.docker.com/desktop/multi-arch/

This enables the user of the action the specify a list of architectures like: `platform: linux/amd64,linux/arm64`

To support different architectures, we run `qemu` to support `docker buildx` virtualisation

Also posting a nice message after an image is uploaded: 

Screenshot 2021-09-23 at 20 50 30

PR used for manual testing of this: https://github.com/Tradeshift/docker-base/pull/387

**Example** 
How to use this feature

```
jobs:
  docker:
    runs-on: self-hosted
    steps:
      - uses: actions/checkout@v2
      - uses: tradeshift/actions-docker@v1
        with:
          password: ${{ secrets.GCLOUD_SERVICE_ACCOUNT_KEY_NOBASE64 }}
          platform: linux/amd64,linux/arm6
```
If you do not specify platform, the builder will default to the old behaviour, so building the docker image with the local platform.

### CODE REVIEW

File: src/docker.ts [hunk 1]
```diff
@@ -71,6 +74,14 @@ async function getSHATag(repository: string): Promise<string> {
   return `${repository}:${res.stdout.trim()}`;
 }
 
+export async function version(): Promise<void> {
```
Comment (JesperTerkelsen) [inline src/docker.ts:77]: This is not really used, but it's useful for getting docker compatibility information, so i will keep it for future debugging easiness.

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+12
```
Comment (RicardoNalesAmato) [inline .nvmrc:1]: can you add this to the .gitignore please?

**Review by RicardoNalesAmato — APPROVED** [review]
👍 nice!

File: .nvmrc (hunk 2 above)
Comment (JesperTerkelsen) [reply .nvmrc:1]: Actually let me delete this file, i was not using it correctly.

**Pull request merged**